docusaurus/android_versioned_docs/version-5.x.x/04-compose/03-general-customization/02-attachment-factory.mdx

```diff
@@ -25,7 +25,7 @@ public open class AttachmentFactory constructor(
 * `content`: Defines the composable function that accepts a `Modifier` and an `AttachmentState` and shows a given attachment component in the message list.
 * `textFormatter`: Lambda function that accepts an attachment and returns a string representation for it.
 
-There are a few examples of default attachment factory implementations, in the [`StreamAttachmentFactories.kt` file](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt):
```

Comment:
Linking while using backtics to highlight the file name render weirdly, plus the file isn't a piece of code either so I don't think it's necessary to do so.